source/extensions/omni.isaac.lab/omni/isaac/lab/sensors/frame_transformer/frame_transformer_data.py

```diff
@@ -15,8 +15,8 @@ class FrameTransformerData:
     """Target frame names (this denotes the order in which that frame data is ordered).
 
     The frame names are resolved from the :attr:`FrameTransformerCfg.FrameCfg.name` field.
-    This usually follows the order in which the frames are defined in the config. However, in
-    the case of regex matching, the order may be different.
+    This does not necessarily follow the order in which the frames are defined in the config due to
```

Comment:
I see. But then how do users get the order they want? Also then it seems a bit strange to ask for a "list" of target_frames if the ordering is not respected from there.

Reply:
We could potentially re-order them, but how would we handle `.*` regex?